helm/values.yaml

```diff
@@ -23,7 +23,7 @@ containerLogs:
     image:
       repository: aws-for-fluent-bit
       tag: 2.31.12.20230911
-      tagWindows: windowsservercore-latest
+      tagWindows: 2.31.12-windowsservercore
```

Comment:
is this temporary? i imagine we will use the same tag for linux and windows eventually with them being multi arch

Reply:
This is not temporary.
ECR repo is same but tag for Windows is different from Linux container images. We will need two tags. We will need to keep values for both tags in sync over time.